.github/workflows/build-graalvm.yml

```diff
@@ -33,17 +33,17 @@ jobs:
           file: ./target/teamscale-upload
       - name: 'Upload Artifact'
         if: ${{ always() }}
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: target-linux
           path: ./target
           retention-days: 5
 
   build-mac-graalvm:
     name: MacOS graalvm Build
-    runs-on: macos-11
+    runs-on: macos-13
```

Comment:
For the reviewer: `macos-11` does not exist anymore as a runner (see [here](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for--private-repositories))